recipes/cppitertools/all/conanfile.py

```diff
@@ -0,0 +1,59 @@
+import os
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.28.0"
+
+
+class CppItertoolsConan(ConanFile):
+    name = "cppitertools"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/ryanhaining/cppitertools"
+    description = "Implementation of python itertools and builtin iteration functions for C++17"
+    topics = ("cpp17", "iter", "itertools")
+    license = "BSD-2-Clause"
+    no_copy_source = True
+
+    settings = "os", "arch", "compiler", "build_type"
```

Comment:
I'm not sure about this. In the future, we should be able to provide to the build-requires the actual values to `settings_target` (not just the ones from the `profile:host`). And, if we _remove_ settings here they won't contain that information.

I'm pretty sure that some build-requirement will try to get information from `self.settings_target.os` and do something. I'm not saying that it will fail, but the risk is higher than if we leave _all_ the settings here.